tests/benchmarks/bench_sched_yield/laze.yml

```
@@ -2,4 +2,5 @@ apps:
   - name: bench_sched_yield
     selects:
       - sw/benchmark
+      - executor-thread
```

I guess this is unrelated?

---

It was needed so that this benchmark compiles for esp, because otherwise the default `executor-single-thread` was selected, that conflicts with the `threading` feature.

But I guess the correct variant is to instead select `sw/threading`, which will down the line result into `executor-thread` being selected for esp.
```suggestion
      - sw/threading
```